### PR TITLE
add r-ramcharts

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Requirements r-piper in side PR #13119 

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dbast** [comment]
Please update/re-create via https://github.com/bgruening/conda_r_skeleton_helper, also spdx and please add conda-forge/r to maintainers

**Comment by FernandezMathieu** [comment]
@conda-forge-admin, please restart ci

File: recipes/r-ramcharts/build.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -o errexit -o pipefail
+if [[ ${target_platform} =~ linux.* ]] || [[ ${target_platform} == win-32 ]] || [[ ${target_platform} == win-64 ]] || [[ ${target_platform} == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  mv DESCRIPTION DESCRIPTION.old
+  grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
+  ${R} CMD INSTALL --build .
```
Comment (dbast) [inline recipes/r-ramcharts/build.sh:7]:
```suggestion
  export DISABLE_AUTOBREW=1
  ${R} CMD INSTALL --build .
```

File: recipes/r-ramcharts/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  version: {{ version|replace("-", "_") }}
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rAmCharts_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rAmCharts/rAmCharts_{{ version }}.tar.gz
+  sha256: 2cdfff2eeb1fc5260e0bac863e18b58db6ada6318b558ab0720fc159e5f92fb1
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
```
Comment (dbast) [inline recipes/r-ramcharts/meta.yaml:16]: Please cleanup by removing all comments, except the cran meta-data at the end.

**Comment by FernandezMathieu** [comment]
We can merge ? @dbast

**Pull request merged**